src/fetcher/historical.py

```
@@ -208,6 +213,13 @@ def fetch(
             except Exception as e:
                 logger.warning(f"Failed to close stream: {e}")
 
+            # Explicitly cleanup COM resources (prevents "Win32 exception releasing IUnknown")
+            if hasattr(self.jvlink, 'cleanup'):
+                try:
+                    self.jvlink.cleanup()
+                except Exception:
+                    pass
```

Ruff (0.14.14)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)